tailwind.config.ts

```diff
@@ -14,7 +14,22 @@ const config: Config = {
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
     },
+    colors: {
+      'white': '#FFFFFF',
+      'primary': '#4880FF',
+      'primary-text': '#FFFFFF',
+      'secondary': '#E2E2E2',
+      'secondary-text': '#7E7E7E',
+      'danger': '#FCD7D4',
+      'danger-text': '#EF3826',
+      'warning': '#FFEDDD',
+      'warning-text': '#FFA756',
+      'success': '#CCF0EB',
+      'success-text': '#00B69B',
```

Comment:
```suggestion
    colors: {
      'white': '#FFFFFF',
      'primary': {
          "bg": '#4880FF', //Soit bg soit DEFAULT
          "text": #FFFFFF
      }
      ...
```
Tu peux faire ça ça sera plus propre si un jour on a du border etc..